--- src/popup.c	2026-05-19 00:24:16.086450399 +0000
+++ src/popup.c	2026-05-19 00:24:16.101139149 +0000
@@ -2282,7 +2282,7 @@
         GdkGrabStatus grab_status;
         grab_status = gdk_seat_grab (seat,
                                      gtk_widget_get_window (popup->window),
-                                     GDK_SEAT_CAPABILITY_ALL_POINTING,
+                                     GDK_SEAT_CAPABILITY_ALL,
                                      TRUE, NULL, NULL, NULL, NULL);
         if (grab_status == GDK_GRAB_SUCCESS) {
             g_warning ("xfce-net-plugin: grab OK, usando button-press");
