--- src/popup.c	2026-05-19 00:57:42.257528367 +0000
+++ src/popup.c	2026-05-19 00:57:42.375808601 +0000
@@ -2169,6 +2169,7 @@
     gtk_window_set_resizable        (GTK_WINDOW (win), FALSE);
     gtk_window_set_type_hint        (GTK_WINDOW (win),
                                      GDK_WINDOW_TYPE_HINT_POPUP_MENU);
+    gtk_window_set_keep_above       (GTK_WINDOW (win), TRUE);
     gtk_widget_set_size_request     (win, -1, -1);
 
     outer_box = gtk_box_new (GTK_ORIENTATION_VERTICAL, 0);
